README.md

```diff
@@ -97,3 +106,8 @@ Using network 'live'.
 Truffle v5.0.25 (core: 5.0.25)
 Node v10.16.3
 ```
+
+## TODO
+
+- Add tests for ChainlinkVRF
+- Add tests for Chainlink Price Feeds
```

Comment:
I'd like to add a few:
- Refactor tests to use `this` instead of defining contracts with `let`
- Use the Chainlink-published mocks for [MockV3Aggregator](https://github.com/smartcontractkit/chainlink/blob/develop/evm-contracts/src/v0.6/tests/MockV3Aggregator.sol) and [VRFCoordinatorMock](https://github.com/smartcontractkit/chainlink/blob/develop/evm-contracts/src/v0.6/tests/VRFCoordinatorMock.sol)
- Upgrade to Solidity 0.6.12 (to show that you don't _have_ to use 0.6.6)